Style

jasonfelice opened the pull request:
### Style Bookstore

- Copy the template style from Zeplin.
  - [x] Styled each component as given in the template.

jasonfelice committed:
Install sass loader

jasonfelice committed:
Style header:
Import monsterrat font from google
Use colors from zeplin template

jasonfelice committed:
Increase the heading font-weight to 700

jasonfelice committed:
Style Book component:
Space information and remove button

jasonfelice committed:
Style buttons in the book component

jasonfelice committed:
Center the main content in the page

jasonfelice committed:
Style Books page

jasonfelice committed:
Align add book form

jasonfelice committed:
Style input fields

jasonfelice committed:
Create a class for buttons

jasonfelice committed:
Update module scss

jasonfelice committed:
Fix StyleLint errors

Ismailco commented on src/App.scss:
```diff
@@ -0,0 +1,37 @@
+  box-sizing: border-box;
+}
+
+body {
```
- [ ] The page should look the same as the Zeplin design, please add everything in the design e.g. `progress circle`, `profile button in the nav bar`, `chapter`...

|Zeplin design|Your design|
|-|-|
|![Screenshot_20220609_052237](https://user-images.githubusercontent.com/25227022/172763569-0c3a3651-7693-4d89-b1aa-3138fbe97c4e.png)|![Screenshot_20220609_051908](https://user-images.githubusercontent.com/25227022/172763517-92155548-1324-4072-a64d-855d4e324ff6.png)|

Ismailco reviewed (changes requested):
### Hi @jasonfelice,

Good job so far 👍🏼 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✔️  No linter errors
✔️ Used Gitflow
✔️ Work documented professionally

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

jasonfelice committed:
Add edit and comment button

jasonfelice committed:
Update favicon

jasonfelice committed:
Create the progress circle

jasonfelice committed:
Create a column for book chapter info

jasonfelice committed:
Space chapter info column

jasonfelice committed:
Fix styleLint error

Lameck1 reviewed (approved):
Hi, @jasonfelice,

You've done well addressing concerns from the previous review 👏🏼.
Tasks for this project are now successfully implemented 💯.

## Highlights
- Professional README ✔️
- Clean code ✔️ 
- Good commit messages ✔️ 

All the best with the next project.

You are free to merge this branch :shipit:
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
**Please, remember to tag me in your question so I can receive the notification.**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

jasonfelice commented:
> Hi, @jasonfelice,
> 
> You've done well addressing concerns from the previous review 👏🏼. Tasks for this project are now successfully implemented 💯.
> 
> ## Highlights
> * Professional README ✔️
> * Clean code ✔️
> * Good commit messages ✔️
> 
> All the best with the next project.
> 
> You are free to merge this branch :shipit: Congratulations! 🎉
> 
> Cheers and Happy coding!👏👏👏
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear. **Please, remember to tag me in your question so I can receive the notification.**
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Thanks
